src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random random = new Random();
```

Comment:
Don't use static methods in your solution. Static methods are generally a bad practice. Instead of a static Random instance, you should create a non-static instance.